web/package-lock.json

```diff
@@ -67,6 +67,7 @@
       "license": "MIT",
       "devDependencies": {
         "@types/node": "^20.11.0",
+        "oazapfts": "^5.1.4",
```

Comment:
Why is that change in here?

Reply:
guess it must have gotten missed somehow in my last PR. not sure why it didn't pick it up then